common/bcp47/timezone.xml

```diff
@@ -176,7 +176,8 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="fotho" description="Faroe Islands" alias="Atlantic/Faeroe Atlantic/Faroe"/>
             <type name="frpar" description="Paris, France" alias="Europe/Paris"/>
             <type name="galbv" description="Libreville, Gabon" alias="Africa/Libreville"/>
-            <type name="gaza" description="Gaza Strip, Palestinian Territories" alias="Asia/Gaza"/>
+            <type name="gaza" description="Gaza Strip, Palestinian Territories" deprecated="true" alias="Asia/Gaza gazastrp"/>
+            <type name="gazastrp" description="Gaza Strip, Palestinian Territories" deprecated="true" alias="Asia/Gaza"/>
```

Comment:
```suggestion
            <type name="gazastrp" description="Gaza Strip, Palestinian Territories" alias="Asia/Gaza"/>
```

Reply:
Whoops! Yes, I meant only the 'gaza' one.

Reply:
would it make sense to promote gaza to a metazone to make the invariant work?